M2/Macaulay2/packages/SpectralSequences.m2

```diff
@@ -2943,52 +2730,50 @@ doc ///
      	  Text
 	     Produces the filtered complex obtained by successively truncating the complex.
 	  Example 
-	    needsPackage "SpectralSequences"
+	    needsPackage "SpectralSequences2"
```

Comment:
Here's an example of "SpectralSequences2" in the package. You could either search & replace all to remove the "2" or, better yet, simply remove the whole line, because examples don't need to reload the package.

Reply:
Oh.  Yes.  Those need to be removed!